src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -28,6 +29,17 @@ trait ResourceRoutes extends SamUserDirectives with SecurityDirectives with SamM
   val resourceService: ResourceService
   val liquibaseConfig: LiquibaseConfig
 
+  def resourceRoutesV3(samUser: SamUser, samRequestContext: SamRequestContext): server.Route =
+    pathPrefix("resources" / "v3") {
```

Comment:
I need to figure out how to get Akka to recognize this as a separate endpoint rather than an arg to `/api/resources/v2`.